rules/crowdstrike_rules/event_stream_rules/crowdstrike_single_ip_allowlisted.yml

```diff
@@ -0,0 +1,250 @@
+AnalysisType: rule
+Filename: crowdstrike_single_ip_allowlisted.py
+RuleID: "Crowdstrike.SingleIpAllowlisted"
+DisplayName: "Crowdstrike Single IP Allowlisted"
+Reports:
+  MITRE ATT&CK:
+    - T1556.009 # Modify Authentication Process: Conditional Access Policies
+Enabled: true
+LogTypes:
+  - Crowdstrike.EventStreams
+Description: A single IP (instead of a CIDR range) was allowlisted. This could indicate a bad actor permitting access from another machine.
```

Comment:
I like how this rule is specific to single IPs being allowlisted, although I think you would also want to know when CIDR blocks were allowed, especially if the subnets were very broad eg 10.0.0.0/8.  Maybe there could be another rule with lower severity for any allowlist change.

Reply:
I added a new rule "Crowdstrike.IpAllowlistChanged" that tracks any changes!